Classes/WDResizableCropOverlayView.swift

```diff
@@ -170,11 +174,19 @@ internal class WDResizableCropOverlayView: WDImageCropOverlayView {
         pointY = pointY > self.superview!.bounds.size.height - border ?
             self.superview!.bounds.size.height - border : pointY
 
-        let heightChange = (pointY - startPoint.y) * resizeMultiplyer.heightMultiplyer
-        let widthChange = (startPoint.x - pointX) * resizeMultiplyer.widthMultiplyer
+        var heightChange = (pointY - startPoint.y) * resizeMultiplyer.heightMultiplyer
+        var widthChange = (startPoint.x - pointX) * resizeMultiplyer.widthMultiplyer
+
+        if lockAspectRatio {
+            let averageChange = (heightChange + widthChange) / 2
```

Comment:
This ensures that the handle currently used for resizing doesn't get too far from the user's touch point.